{
  "comments": [
    {
      "unresolved": false,
      "key": {
        "uuid": "dc4ac208_c3aa4415",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 1
      },
      "lineNbr": 0,
      "author": {
        "id": 1000056
      },
      "writtenOn": "2021-02-03T10:44:09Z",
      "side": 1,
      "message": "I\u0027m getting the same error on my PC",
      "revId": "fa46dd8a13b3f206a2ac8b1857f3b1ecb627f825",
      "serverId": "e32f8df8-3db0-4f76-a6c6-fe9d6e69dd68"
    }
  ]
}